.github/workflows/deploy.yml

```diff
@@ -25,6 +25,7 @@ jobs:
       - name: Checkout Package
         uses: actions/checkout@v2
         with:
+          ref: ${{ github.ref_name }}
```

Comment:
I believe these are redundant as the default value should be the SHA of the commit triggering this workflow. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Reply:
The default value would be the branch name that triggered the workflow so I do not think this is redundant.